gem/templates/chhaajja/videos/videos_main_block.html

```diff
@@ -0,0 +1,174 @@
+                               <div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_content">
+										<h5 class="righth5">What is Lorem Ipsum?</h5>
+										<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+									</div>
+								</div>
+								<div  class="clear-all"></div>
+								<!--<div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_img">
+										<img src="images/Side-img1.jpg"  class="img-responsive" alt="">
+									</div>
+								</div>
+                               <div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_content">
+										<h5 class="righth5">What is Lorem Ipsum?</h5>
+										<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+									</div>
+								</div>
+								<div  class="clear-all"></div>-->
+								
+                                <div class="sidebar_title1 ">
+                                    <h3> <img src="{% static 'images/Sym2.png' %}"  class="" alt=""> Arre sun na, ye bhi hai!</h3>
+                                </div>
+								<div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_img">
+										<img src="{% static 'images/Side-img3.jpg' %}"  class="img-responsive" alt="">
+									</div>
+								</div>
+                               <div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_content">
+										<h5 class="righth5">What is Lorem Ipsum?</h5>
+										<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+									</div>
+								</div>
+								<div  class="clear-all"></div>
+								<div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_img">
+										<img src="{% static 'images/Side-img4.jpg' %}"  class="img-responsive" alt="">
+									</div>
+								</div>
+                               <div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_content">
+										<h5 class="righth5">What is Lorem Ipsum?</h5>
+										<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+									</div>
+								</div>
+								<div  class="clear-all"></div>
+								<!--<div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_img">
+										<img src="images/Side-img3.jpg"  class="img-responsive" alt="">
+									</div>
+								</div>
+                               <div class="col-md-6  col-xs-6 col-sm-6">
+									<div class="blog_list_content">
+										<h5 class="righth5">What is Lorem Ipsum?</h5>
+										<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+									</div>
+								</div>
+								<div  class="clear-all"></div>-->
+								
+								</div>
+                            </aside>
+							
+							
+							
+							<div style="clear:both;"></div>
+                            <aside class="right_widget recent_widget commentsection1">
```

Comment:
We already have commenting functionality available on our platform, it would just need styling. It is already integrated with wagtail admin and managed from there. You can find more info here https://github.com/praekeltfoundation/molo.commenting

This commenting module is used across all sites and can be added to any page.